daisy_workflows/image_build/windows/windows-server-2022-dc-core-uefi-byol.wf.json

```diff
@@ -0,0 +1,107 @@
+{
+  "Name": "windows-2022-dc-core-uefi-byol",
+  "Vars": {
+    "install_disk": "install-disk",
+    "media": {
+      "Required": true,
+      "Description": "GCS or local path to Windows Server ISO."
+    },
+    "pwsh": {
+      "Required": true,
+      "Description": "GCS or local path to PowerShell v7+ installer"
+    },
+    "dotnet48": {
+      "Required": true,
+      "Description": "GCS or local path to Microsoft .NET Framework 4.8 offline installer"
+    },
+    "cloudsdk": {
+      "Required": true,
+      "Description": "GCS or local path to Cloud SDK installer"
+    },
+    "description": {
+      "Value": "Microsoft, Windows Server, 2022 Datacenter Core, Server Core, x64 built on ${TIMESTAMP}, supports Shielded VM features"
+    },
+    "family": {
+      "Value": "windows-2022-core-byol",
+      "Description": "Desired image family of the output image."
+    },
+    "google_cloud_repo": {
+      "Value": "stable",
+      "Description": "Google Cloud repo to retrieve packages from during the build."
+    },
+    "name": {
+      "Value": "windows-server-2022-dc-core-byol-v${TIMESTAMP}",
+      "Description": "The name of the output image."
+    },
+    "project": {
+      "Value": "${PROJECT}",
+      "Description": "The GCP project to create and store the image in."
+    },
+    "updates": {
+      "Value": "",
+      "Description": "GCS or local filesystem location containing Windows update files."
+    },
+    "workflow_root": {
+      "Value": "/workflows",
+      "Description": "Root of github workflows, defaults to /workflows in the container."
+    }
+  },
+  "Steps": {
+    "windows-build": {
+      "Timeout": "4h",
+      "IncludeWorkflow": {
+        "Path": "./windows-build-uefi.wf.json",
+        "Vars": {
+          "install_disk": "${install_disk}",
+          "install_disk_size": "32",
```

Comment:
MS lists 32GB as the absolute min disk space required. How much disk space is free for a server core installation?

With 20h2, I increased the disk size to 40GB after finding we were too close for comfort.

https://docs.microsoft.com/en-us/windows-server/get-started/hardware-requirements#storage-controller-and-disk-space-requirements

Reply:
9.1 GB used, 22.8 GB free -- that's not too bad, but I guess it's better to increase to 40 GB as well.